builders/deploy/dapp.md

```diff
@@ -130,38 +144,31 @@ Finally, [generate and edit](/builders/build/customize/customizing-chain-specs/#
 
 Now, you can upload your custom raw specification file by selecting the **Custom** template and adding your JSON specification file.
 
-![Upload a custom raw specification file to the Tanssi dApp.](/images/builders/deploy/dapp/dapp-4.png)
+![Upload a custom raw specification file to the Tanssi dApp.](/images/builders/deploy/dapp/dapp-7.png)
 
 !!! note
     The size of a raw chain specifications file should not exceed 2MB.
 
-## Check Your Balances {: #check-balances }
-
-After configuring your Appchain, the dApp will require you to connect your wallet to check your balances. Provided that you have received enough DANCE and UNIT tokens, you can go ahead.
+## Check Balances {: #check-balances }
 
-### Connect Your Wallet to the DApp {: #connect-wallet }
-
-To continue the process, you'll be required to connect your wallet. Depending on the wallet you're using, your wallet may pop up when first navigating to the Tanssi dApp. If so, feel free to follow the prompt, select the accounts you want to interact with, and connect your wallet.
-
-Otherwise, to get started, you can:
-
-1. Click **Connect Wallet**
-2. Choose your account from the dropdown
-3. You'll be prompted to sign a message, which will log you into the Tanssi dApp. Go ahead and sign the message
-
-![Click on the Connect Wallet button to connect your wallet to the Tanssi dApp.](/images/builders/deploy/dapp/dapp-5.png)
+Next, you'll need to verify that you have sufficient balances of DANCE and UNIT tokens. If you don't, you can press **Request Tokens** and complete the following login with GitHub or Google. You'll need to complete a few quick onboarding questions, and then you'll be able to press **Request Tokens** again, and they will be delivered to your connected wallet.
 
-Once connected, you'll see your address in the top-right corner. If you've connected multiple accounts and would like to switch accounts, you can click on your address and choose an account from the dropdown menu.
+![Request tokens](/images/builders/deploy/dapp/dapp-8.png)
 
-### Balance Verification {: #balance-verification }
+If you're setting up a dedicated Appchain, you'll need to manually request the necessary tokens via a [form on the Tanssi network website](https://www.tanssi.network/claim-dance-tokens){target=\_blank}, and you'll receive the necessary tokens within one business day. The required minimum balances to launch an Appchain are as follows: 
 
-Once you have connected your wallet to the dApp, the **Balance Verification** section will show your DANCE and UNIT token balances. If you meet the requirements, it results in a green checkmark next to each balance, while failing to meet them is indicated by a red X.
-
-If you have already reserved your Appchain ID, you can toggle the switch at the bottom of the **Balance Verification** page, and your balance requirements will change.
+=== "Snap Appchain"
+    |              Chain               | Balance Required |
```

Comment:
These tables can be formatted with the markdown table formatter extension in vs code